<?xml version="1.0"?>

<launch>

  <!-- <arg name="world/map" default="tutorial_1.world"/> -->

  <arg name="map_frame" default="map"/>
  <arg name="camera_frame" default="cf1/camera_link"/>
  <arg name="base_frame" default="cf1/base_link"/>
  <arg name="world/map" default="crazyflie9_apartment.world"/>


  <group>
    <include file="$(find dd2419_launch)/launch/base.launch" />
    <!-- <include file="$(find dd2419_simulation)/launch/simulation.launch" /> -->

    <include file="$(find milestone3)/launch/multi_road_sign_pose.launch" />
    <node type="Localization_V2.py" name="Localization" pkg="milestone3" output="screen" />
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="map_to_odom" args="-0.5 -0.5 0 0 0 0 1 map cf1/odom" /> -->

    <node type="map_rviz_transmitted.py" name="occupancy_node" pkg="milestone2" output="screen" />
    <node type="displaymapmarkers" name="displaymapmarkers" pkg="milestone2" args="$(find milestone3)/../worlds_json/$(arg world/map).json"/>
    <node type="displaymapsigns" name="displaymapsigns" pkg="milestone2" args="$(find milestone3)/../worlds_json/$(arg world/map).json"/>

    <node pkg="milestone2" type="clear_checkpoint_service" name="clear_checkpoint_service" />
    <node type="path_planning_server.py" name="path_planning_server" pkg="milestone3" output="screen" />

    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find milestone3)/include/milestone3.rviz" output="screen" />

    <node pkg="tf2_ros" type="static_transform_publisher" name="cam_to_drone" args="0.01 0 0.02 -1.57 0 -1.57 $(arg base_frame) $(arg camera_frame)" />



  </group>

</launch>

